src/components/Person/Person.jsx

```diff
@@ -1 +1,29 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const SEX_MALE = 'm';
+
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const partnerType = (sex === SEX_MALE) ? 'wife' : 'husband';
+
+  const partnerText = isMarried
+    ? `${partnerName} is my ${partnerType}`
+    : 'I am not married';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+
+      {!!age && <p className="Person__age">{`I am ${age}`}</p>}
```

Comment:
Move paragraph to the new line, let's make code clearer
```suggestion
      {!!age && (
        <p className="Person__age">
          {`I am ${age}`}
        </p>
      )}
```